frontend/sample.env

```diff
@@ -1,3 +1,13 @@
 # Fix for https://stackoverflow.com/questions/70374005/invalid-options-object-dev-server-has-been-initialized-using-an-options-object
 DANGEROUSLY_DISABLE_HOST_CHECK=true
-REACT_APP_KEYCLOAK_URL=https://auth.star.vote:8443/realms/STARVotingDev/protocol/openid-connect
\ No newline at end of file
+REACT_APP_KEYCLOAK_URL=https://auth.star.vote:8443/realms/STARVotingDev/protocol/openid-connect
+
+FF_ELECTION_ROLES=false
+FF_MULTI_WINNER=false
+FF_METHOD_STAR_PR=false
+FF_METHOD_RANKED_ROBIN=false
+FF_METHOD_APPROVAL=false
+FF_METHOD_PLURALITY=false
+FF_METHOD_RANKED_CHOICE=false
+FF_CANDIDATE_DETAILS=false
+FF_MULTI_RACE=false
```

Comment:
Did this work in the build process? I think React env variables have to be prefixed with "REACT_APP_" https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
Updated to use REACT_APP_